### DISCUSSION TITLE
This library is awesome, is there a Java equivalent?

### DISCUSSION THREAD

**Discussion by bpatchin1** [aws/amazon-redshift-python-driver#221] [Q&A] [answered]
Hi!

I just want to say that this library is awesome and it is extremely useful for python applications that are connecting to Redshift. Wondering if there is an equivalent in Java (redshift data api + java sdk v2) don't really meet our needs (specifically connecting to database using idp + username and password). 

Does anyone know if there is an equivalent package in Java?

Thanks!

**Comment by Brooke-white** [comment, marked as answer]
Hi @bpatchin1  -- My apologies for the late response. Please checkout the Amazon Redshift JDBC driver: https://github.com/aws/amazon-redshift-jdbc-driver
